ipsec-vpn/hier/usr/share/untangle/bin/ipsec-restart-l2tp

```diff
@@ -0,0 +1,15 @@
+#!/bin/dash
+
+# Restart the xlt2pd daemon when VPN is Enabled
+# Stop if Disabled
+
+if [ "$1" = "VPN_ENABLED" ]; then
+    systemctl reset-failed xl2tpd 2>&1 1>> /dev/null
+    systemctl restart xl2tpd 2>&1 1>> /dev/null
+
+else
+    systemctl stop xl2tpd 2>&1 1>> /dev/null
```

Comment:
add this command before 
systemctl reset-failed xl2tpd 2>&1 1>> /dev/null

Reply:
Done